pending-documentation.md

```diff
@@ -197,6 +197,50 @@ This will rewrite I to . if and only if the state cell contains I + 1.
 
 Note that in the case of Set and Map, one guarantee is that K1, K2, K3, and K4 represent /distinct/ elements. Pattern matching fails if the correct number of distinct elements cannot be found.
 
+## Set Variables
+
+### Motivation
+
+Set variables were introduced as part of Matching Mu Logic, the mathematical foundations for K.
+In Matching Mu Logic, terms evaluate to sets of values. This is useful for both capturing partiality (as in `3/0`) and
+capturing non-determinism (as in `3 #Or 5`).
+Consequently, symbol interpretation is extended to have a collective interpretation over sets of input values.
+
+Usually, K rules are given using regular variables, which expect that the term they match is both defined and has a
+unique interpretation.
+
+However, it is sometimes useful to have simplification rules which work over any kind of pattern, be it undefined or
+non-deterministic.  This behavior can be achieved by using set variables to stand for any kind of pattern.
+
+### Syntax
+
+Any variable prefixed by `@` will be considered a set variable.
+
+### Example
+
+Below is a simplification rule which motivated this extension:
```

Comment:
This is an interesting rule but it was not what motivated the extension.  There are many and complex reasons for the extension, which we do not need to get into here.  I would remove the "which motivated the extension" part.

Reply:
These are the rules which motivated the extension to K (perhaps not the same extension to matching logic). These are rules we need to get the Haskell backend to prove several of the single-opcode KEVM proofs. I think as pending documentation for K, it's appropriate to use K rules motivations.